src/OrleansAzureUtils/OrleansAzureUtils.csproj

```diff
@@ -21,7 +28,12 @@
   <ItemGroup>
     <ProjectReference Include="..\Orleans.Core.Abstractions\Orleans.Core.Abstractions.csproj" />
     <ProjectReference Include="..\Orleans.Core\Orleans.Core.csproj" />
+    <ProjectReference Include="..\Orleans.Clustering.AzureStorage\Orleans.Clustering.AzureStorage.csproj" />
     <ProjectReference Include="..\Orleans.Runtime\Orleans.Runtime.csproj" />
     <ProjectReference Include="..\OrleansProviders\OrleansProviders.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Utilities\" />
```

Comment:
This is probably a remnant of some deleted code?

Reply:
It is because of the linked file which resides in the Utilities folder, VS added it.